cli/clients.go

```diff
@@ -72,7 +72,7 @@ var cmdClients = []cobra.Command{
 				logJSONCmd(*cmd, l)
 				return
 			}
-			t, err := sdk.Client(args[0], args[1], args[2])
+			t, err := sdk.Client(args[0], args[1], args[2], false)
```

Comment:
Please add options CLI for to retreive with roles  like we can have args `--roles`